mobile-app/cypress/integration/functional/wallet/balances/balances.spec.ts

```diff
@@ -1,19 +1,31 @@
+import { WhaleApiClient } from '@defichain/whale-api-client'
+import BigNumber from 'bignumber.js'
+
 export interface BalanceTokenDetail {
   symbol: string
   name: string
   amount: string | number
+  activePrice?: string
 }
 
 context('Wallet - Balances', () => {
+  let whale: WhaleApiClient
+
   before(function () {
     cy.createEmptyWallet(true)
     cy.sendDFItoWallet().wait(3000)
     cy.getByTestID('header_settings').click()
     cy.getByTestID('bottom_tab_balances').click()
+    const network = localStorage.getItem('Development.NETWORK')
+    whale = new WhaleApiClient({
```

Comment:
can you help to refactor this into a command 🙏? saw a few repeated usage on other .spec too